src/main/java/edu/ucsb/cs156/gauchoride/entities/Shift.java

```diff
@@ -0,0 +1,26 @@
+package edu.ucsb.cs156.gauchoride.entities;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.AccessLevel;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Builder
+@Entity(name = "shift")
+public class Shift {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+  private String day;
+  private String shift;
```

Comment:
(we already discussed this in section today, but...) in the backend it is probably better to have LocalTime start and LocalTime end and we can display this as a shift in the table.  Keeping the day a string (or maybe an integer 0 through 6) for now is probably ok since we want to work with weekly shift schedules.

Reply:
In my new commit, I use LocalTime for shiftStart and shiftEnd.